en/Editing and formatting/Folding.md

```diff
@@ -0,0 +1,16 @@
+Learn how to get a better overview of large notes by using _folding_ to hide parts of the note. Folding is useful when creating outlines for your notes and when you want to focus on what you're working on at the moment.
+
+You can fold headings and indented lists by hovering the mouse cursor over the section you want to fold, and then selecting the arrow on the left. Folded sections show an arrow regardless of if you hover it or not.
+
+Folding is turned on by default. To turn off folding, open **Settings** → **Editor**, and then turn off **Fold indent** or **Fold heading**, depending on your needs.
+
+To toggle all sections at the same time, use the following commands:
+
+- To collapse all sections, open the [[Command palette]], and then select **Fold all headings and lists**.
+- To expand all sections, open the [[Command palette]], and then select **Unfold all headings and lists**.
+
+> [!tip]
+> If you prefer to fold using your keyboard, you can assign [[Use hotkeys|hotkeys]] to the **Fold less** and **Fold more** commands.
+>
+> - **Fold less** unfolds the section at the text cursor.
+> - **Fold more** folds the previous section closest to the text cursor.
```

Comment:
```suggestion
> - **Fold more** folds the section closest to the text cursor.
```

Reply:
Technically, this is incorrect isn't it?

In the example below, even though Heading 2 is closest to the cursor, it's Heading 1 that gets folded.

```
## Heading 1

Lorem ipsum

This sentence is ▋

## Heading 2
```

Reply:
So, it depends how we're defining "section" I guess. If we're using the Obsidian notion of a "section" then the entire paragraph—including the heading—is all part of the same section.

The fold more and fold less commands also work in lists:

```
- Lorem ipsum
  - This sentence is ▋
- Donec faubicus
```

Reply:
> If we're using the Obsidian notion of a "section" then the entire paragraph—including the heading—is all part of the same section.

Yes. That's how I see it as well. I realize now that I meant "closest previous heading that defines the section", but that's a horrible sentence 😅

Moving away from describing it in terms of distance from the cursor, how about this then?

```suggestion
> - **Fold more** folds the section or list that contains the text cursor.
```

Reply:
I like option 1! That seems clear enough to me.